Package.swift

```diff
@@ -20,7 +20,7 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/pinterest/PINOperation.git", from: "1.2.0"),
+        .package(url: "https://github.com/pinterest/PINOperation.git", from: "1.2.2"),
```

Comment:
Same thing as mentioned in [this comment](https://github.com/pinterest/PINCache/pull/327/files#r1585229928).